import pyodbcfrom dotenv import load_dotenvimport osload_dotenv()server = os.getenv('DATABASE_SERVER')database = os.getenv('DATABASE_NAME')username = os.getenv('DATABASE_USERNAME')password = os.getenv('DATABASE_PASSWORD')def connect_to_vario(produkt_id):    try:        conn = pyodbc.connect(            f"DRIVER={{ODBC Driver 17 for SQL Server}};"            f"SERVER={server};"            f"DATABASE={database};"            f"UID={username};PWD={password}"        )        row_dict = {}        cursor = conn.cursor()        cursor.execute("SELECT produkt, u_NazevCM, u_IDPacienta, u_typCM, Referent, u_Sterilita, u_Typ_cementace, u_Strana, u_Datum_operace, u_StavCM, u_Nemocnice, u_Lekar, u_Indikace, u_Predpis FROM dbo.Katalog WHERE produkt = ?;", produkt_id)        row = cursor.fetchone()        if row:            print("✅ Read access confirmed.")            row_dict = dict(zip([column[0] for column in cursor.description], row))        else:            print("⚠️ Connected, but no data found.")        conn.close()        return row_dict    except Exception as e:        print("❌ Connection or read failed:", e)        return Noneif __name__ == '__main__':    connect_to_vario("CM250118")